index.spec.js

```diff
@@ -842,6 +842,43 @@ describe("synchronous-promise", function () {
       expect(capturedError).property("errors").to.contain("123");
     });
 
+    it("should reject with AggregateError if supported", function () {
```

Comment:
minor: suggest putting this test in its own `describe` block and having the mocked `AggregateError` set up in a `beforeEach` and torn down in an `afterEach` -- covers the situation where the test, for some reason, fails.

Reply:
I'll give that a go, the only tiny problem I see is that this would never test the code without the `AggregateError` available.

Reply:
Nevermind, if I nest the beforeEach in the describe it is scoped to that test.

Reply:
I would call the test "static any (browsers with AggregateError)", do you have a better name for that?